aws/resource_aws_rds_cluster_test.go

```diff
@@ -2250,6 +2250,22 @@ resource "aws_rds_cluster" "test" {
 `, rName, deletionProtection)
 }
 
+func testAccAWSRDSClusterConfig_GlobalEngineMode(rName string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "aws_rds_cluster" "test" {
+  depends_on		   = ["aws_db_subnet_group.dbsubnet"]
```

Comment:
Instead of using `depends_on` here, a direct reference can be used below instead:

```hcl
  db_subnet_group_name = "${aws_db_subnet_group.dbsubnet.name}"
```